dsa-23au/java-dsa/sorting/src/main/java/dev/codewithfriends/NearestNeighbor.java

```diff
@@ -62,5 +64,32 @@ public static Point findNearestNeighbor(Point[] allPoints, Point target) {
 
         return closestPoint;
     }
+
+    public static final int NEIGHBORHOOD_SIZE = 8_350_000;
+    public static final int MAX_X = 1_000_000;
+    public static final int MAX_Y = 1_000_000;
+
+    public static Random rand = new Random();
+    static Point[] neighborhood;
+    static Point target;
+
+    public static void main(String[] args) {
+
```

Comment:
line 79: at the end of your for loop, you increment i plus 1 by +=. Is this the same as doing i++?